src/quafu/dagcircuits/circuit_dag.py

```diff
@@ -3,7 +3,7 @@
 
 from quafu.elements.element_gates import *
 from quafu.elements.quantum_element import Barrier, Delay, Measure, XYResonance
-from quafu.pulses.quantum_pulse import GaussianPulse, RectPulse, FlattopPulse
+from quafu.elements.quantum_element.pulses.quantum_pulse import GaussianPulse, RectPulse, FlattopPulse
```

Comment:
consider adding imports in `__init__.py` to avoid too many `.`

for example, you might consider add `GaussianPulse` in `__init__.py` under `quafu.elements` directory

simply

```python
from .pulses.quantum_pulse import GaussianPulse, RectPulse, FlattopPulse
```

Then you could use `from quafu.elements import GaussianPulse, RectPulse, FlattopPulse`